docs/guides/development-guides/how-to-show-and-hide-a-split-view-pane-with-mvvm.md

```diff
@@ -3,17 +3,304 @@ id: how-to-show-and-hide-a-split-view-pane-with-mvvm
+            </Button>
+            <TextBlock VerticalAlignment="Center" Margin="15, 0, 0, 0" FontSize="25" Text="Settings"/>
+        </StackPanel>
+    </SplitView.Pane>
+    <TextBlock HorizontalAlignment="Center" VerticalAlignment="Center" Text="MainWindow" FontSize="25"/>
+</SplitView>
+```
+
+## Binding the SplitView's `IsPaneOpen` property to a ViewModel's property.
+Fist of all, create a boolean property with a private backing field in the ViewModel which will contains the boolean value. Now, by using Reactive UI, make it notify every change to the field to the UI. Remember to initialize the backing property with the value you need in the VewModel's constructor.
 :::info
-This technique uses a complex **binding path** to locate the parent view model for the 
+By default in an MVVM project each ViewModel inherits from a ViewModelBase which inherits from the ReactiveObject class. By doing so in every ViewModel there will be the necessary methods to notify the UI. For more informations see [here](https://docs.avaloniaui.net/docs/concepts/reactiveui/reactive-view-model).
 :::
+``` C#
+private bool _isSplitViewPaneOpen;
+public bool IsSplitViewPaneOpen
+{
+    get => this._isSplitViewPaneOpen;
+    set
+    {
+        this.RaiseAndSetIfChanged(ref this._isSplitViewPaneOpen, value);
+    }
+}
+public MainWindowViewModel()
+{
+    this._isSplitViewPaneOpen = false;
+}
+```
+Now that we have the property with its backing field initialized, we can bind it to the SplitView's property: add the following code to the SplitView tag.
+``` xml
+IsPaneOpen="{Binding IsSplitViewPaneOpen}"
+```
+You should now have the following configuration:
+``` xml
+<SplitView PanePlacement="Right" DisplayMode="CompactInline" IsPaneOpen="{Binding IsSplitViewPaneOpen}">
+```
+
+## Configuring the button
+At this point all we need to do is make the button open and close the SplitView's pane when clicked.
+To do so create a public property of type ICommand in the ViewModel with just the "getter method".
+``` C#
+public ICommand ChangeSplitViewPaneStatusCommand { get; }
+```
+And initialize it in the constructor.
+``` C#
+this.ChangeSplitViewPaneStatusCommand = ReactiveCommand.Create(() =>
+{
+    this.IsSplitViewPaneOpen = !this.IsSplitViewPaneOpen;
+});
+```
+In this example I will be using a very simple logic: if the value is true it becomes false, and if it's false it becomes true.
+<br>
+You should now have the following code in the ViewModel:
+
+``` C#
+public class MainWindowViewModel : ViewModelBase
+{
+    private bool _isSplitViewPaneOpen;
+    public bool IsSplitViewPaneOpen
+    {
+        get => this._isSplitViewPaneOpen;
+        set
+        {
+            this.RaiseAndSetIfChanged(ref this._isSplitViewPaneOpen, value);
+        }
+    }
 
-TO DO
+    public ICommand ChangeSplitViewPaneStatusCommand { get; }
 
-<img src={SplitViewExpanderScreenshot} alt=""/>
+    public MainWindowViewModel()
+    {
+        this._isSplitViewPaneOpen = false;
```

Comment:
I disagree. I think that for someone who doesn't have much experience with C# it's better to directly see the dafult value being assigned. I added that line also because someone may need the Pane to be shown when starting the app/visualizing the view, and for someone new to Avalonia may be better to know what to modify from the start

Reply:
I would suggest then adding a comment in the code explaining that

Reply:
Done!